packages/generator/src/edmx-to-vdm/v4/operation.ts

```diff
@@ -120,17 +120,31 @@ export function filterAndTransformOperations(
   operations: EdmxOperation[],
   isBound: boolean
 ): EdmxJoinedOperation[] {
-  const filteredByBoundOperations = operations.filter(
-    operation => (operation.IsBound.toLowerCase() === 'true') === isBound
+  const [withoutOperations, withOperations] = splitOperationsByMissingImports(
+    operationImports,
+    operations
   );
 
-  const [withoutOperation, withOperation] = splitMissingOperation(
-    operationImports,
-    filteredByBoundOperations
+  const relevantOperations = withOperations.filter(
+    operation => operation.IsBound === isBound
   );
 
+  const validBoundOperations =
+    validateBoundOperationParameters(relevantOperations);
+
+  validateOperationImports(
+    withoutOperations,
+    withoutOperations[0]?.operationType
+  );
+
+  return validBoundOperations;
```

Comment:
[req] The naming is very confusing here, because in the case of `isBound === false` those are not bound operations that we return. (see comment above)

```suggestion
  return validBoundOperations;
```